pkg/shell/credentials.jsx

```diff
@@ -260,21 +260,21 @@ const KeyPassword = ({ currentKey, change, setDialogError }) => {
                 </HelperTextItem>
             </HelperText>}
             <FormGroup label={_("Password")} fieldId={(currentKey.name || currentKey.comment) + "-old-password"} type="password">
-                <TextInput type="password" id={(currentKey.name || currentKey.comment) + "-old-password"} value={oldPassword} onChange={setOldPassword} />
+                <TextInput type="password" id={(currentKey.name || currentKey.comment) + "-old-password"} value={oldPassword} onChange={(_event, value) => setOldPassword(value)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230520-070544-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)